tsl/src/fdw/data_node_scan_plan.c

```diff
@@ -504,6 +507,9 @@ data_node_scan_add_node_paths(PlannerInfo *root, RelOptInfo *hyper_rel)
 #ifdef TS_DEBUG
 		if (ts_debug_optimizer_flags.show_rel)
 			tsl_debug_log_rel_with_paths(root, data_node_rel, (UpperRelationKind *) NULL);
+#endif
+#if PG15_GE
+		hyper_rel->live_parts = bms_add_member(hyper_rel->live_parts, i);
 #endif
```

Comment:
It looks like this will set the `live_parts` bit also for dummy paths. Do you really need to scan dummy paths or should you exclude them as well, that is, move this statement to inside the condition above?

Reply:
I excluded them as well.